services/pull/merge.go

```diff
@@ -29,8 +30,88 @@ import (
 	"code.gitea.io/gitea/modules/timeutil"
 	asymkey_service "code.gitea.io/gitea/services/asymkey"
 	issue_service "code.gitea.io/gitea/services/issue"
+
+	"github.com/unknwon/com"
```

Comment:
rm ?

Reply:
I have made a PR: 

* https://github.com/go-gitea/gitea/pull/19284 , keeps the same rendering behavior as legacy `com.Expand`, doesn't block users if there is a template error in some cases.

However, lunny wants to show 500 error page to users if the template can not be rendered correctly.

I believe we should show the partially rendered template to users even if the user template is not good. Because in some case (for example, render the merge message), even the template is not good, we shouldn't frighten end users by 500 errors. We can just show what they write in the template, users can know how to fix.